### PR TITLE
Update DB page to not include only MySQL and PG

### PR DESCRIPTION
**Pull request by ardeche07**
@benarent can you please give this a quick review?  Made the smallest possible change to get this page not terribly inaccurate.

### CODE REVIEW

File: pages/how-it-works/database-access/index.mdx [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-h1: Secure Database Access for PostgreSQL and MySQL
-title: How to access PostgreSQL and MySQL behind NAT
-description: How to implement identity-based database access to MySQL and PostgreSQL databases running behind NAT
-headerTitle: Secure Database Access for PostgreSQL and MySQL
-headerDescription: Teleport provides an identity-aware access proxy with native support for MySQL and PostgreSQL protocols. This chapter explains how database access works.
+h1: Secure Database Access for SQL & NoSQL Databases
+title: How to access SQL & NoSQL Databases
+description: How to implement identity-based database access for SQL & NoSQL databases running in any environment.
+headerTitle: Secure Database Access for SQL & NoSQL Databases
+headerDescription: Teleport provides an identity-aware access proxy with native support for SQL & NoSQL databases such as MySQL, PostgreSQL, MongoDB, CockroachDB any many more. This chapter explains how database access works.
```
Comment (benarent) [inline pages/how-it-works/database-access/index.mdx:8]:
```suggestion
headerDescription: Teleport provides an identity-aware access proxy with native support for SQL & NoSQL databases such as MySQL, PostgreSQL, MongoDB, CockroachDB any many more. This chapter explains how database access works.
```

**Review by benarent — APPROVED** [review]

File: pages/how-it-works/database-access/index.mdx (hunk 1 above)
Comment (C-STYR) [reply pages/how-it-works/database-access/index.mdx:8]: @benarent @ardeche07  ...CockroachDB _and_ many more.

**Review by C-STYR — CHANGES_REQUESTED** [review]
Looks good to me once typo is fixed that I noted in above comment.

File: pages/how-it-works/database-access/index.mdx (hunk 1 above)
Comment (benarent) [reply pages/how-it-works/database-access/index.mdx:8]: fixed

**Comment by benarent** [comment]
@C-STYR PTAL

**Review by C-STYR — APPROVED** [review]

**Comment by C-STYR** [comment]
@benarent please double check the typo - doesn't look like it was committed

File: pages/how-it-works/database-access/index.mdx (hunk 1 above)
Comment (ardeche07) [inline pages/how-it-works/database-access/index.mdx:8]:
```suggestion
headerDescription: Teleport provides an identity-aware access proxy with native support for SQL & NoSQL databases such as MySQL, PostgreSQL, MongoDB, CockroachDB and many more. This chapter explains how database access works.
```

**Comment by ardeche07** [comment]
@C-STYR I committed the change for that type. Good to go.

**Comment by ardeche07** [comment]
@C-STYR can this one go too?

**Review by C-STYR — APPROVED** [review]

**Pull request closed without merging**